modular_skyrat/modules/synths/code/surgery/mechanic_steps.dm

```diff
@@ -2,16 +2,21 @@
 /datum/surgery_step/cut_wires
 	name = "cut wires"
 	implements = list(
-		TOOL_WIRECUTTER		= 100,
-		TOOL_SCALPEL 		= 75,
+		TOOL_WIRECUTTER = 100,
+		TOOL_SCALPEL = 75,
 		/obj/item/knife	= 50,
-		/obj/item				= 10) // 10% success with any sharp item.
+		/obj/item = 10,
```

Comment:
Does it really work like this? That feels scuffed, but I guess if it works, it works.

Reply:
Welcome to surgery code.